src/main/scala/uk/gov/nationalarchives/tdr/api/auth/AuthorisationTags.scala

```diff
@@ -145,3 +146,17 @@ object ValidateHasChecksumMetadataAccess extends SyncAuthorisationTag {
     }
   }
 }
+
+object ValidateHasFFIDMetadataAccess extends SyncAuthorisationTag {
+  override def validateSync(ctx: Context[ConsignmentApiContext, _]): BeforeFieldResult[ConsignmentApiContext, Unit] = {
+    val token = ctx.ctx.accessToken
+    val fileFormatAccess = token.backendChecksRoles.contains(fileFormatRole)
+
+    if (fileFormatAccess) {
+      continue
+    } else {
+      val tokenUserId = token.userId.getOrElse("")
+      throw AuthorisationException(s"User '$tokenUserId' does not have permission to update file format metadata")
```

Comment:
The error message says "does not have permission to update the metadata", but it's being used in a query field which isn't updating anything. Should we have a separate permission about fetching file data?

Reply:
Yeah good point. I've added in a `client_file_metadata` role to control access.